app/graph/types/project_media_type.rb

```diff
@@ -85,9 +85,14 @@
 
   field :annotations_count do
```

Comment:
Please add a test to this field on `test/controllers/graphql_controller_test.rb`